apps/opencs/model/world/refidadapterimp.cpp

```diff
@@ -942,7 +942,8 @@ QVariant CSMWorld::NpcAttributesRefIdAdapter::getNestedData (const RefIdColumn *
     const ESM::NPC::NPDTstruct52& npcStruct = record.get().mNpdt52;
 
     if (subColIndex == 0)
-        return subRowIndex;
+        return subRowIndex;  if (subRowIndex < 0 || subRowIndex > 2 || subColIndex < 0 || subColIndex > 2)
```

Comment:
What are you trying to do here?

Reply:
typo .. github integrated editor, I now pushing without this unwanted change